web-app/src/app/_objects/lapData.ts

```diff
@@ -0,0 +1,8 @@
+export class LapData {
+    lapNumber = -1;
+    lapTime: 'stringTime';
+    totalPowerIn: -1;
```

Comment:
use `=`, not `:`, `:` just tells us the type of the variable, not assigning the value
Same goes for the rest.